test/slack_payloads.expected

```diff
@@ -557,9 +565,11 @@ will notify #all-push-events
   ]
 }
 ===== file ../mock_payloads/status.success_test_non_main_branch.json =====
-will notify #default
+looking up slack user by email mail@example.org
+{ "email": "mail@example.org" }
+will notify #mock_id
```

Comment:
I see what you mean, the problem is that the "will notify xxx" is determined from the channel you provide to the slack api. For a direct message you need to pass the id of the user. So I thought it might be a bit misleading if we store the email in the `id` value

An alternative that I can suggest is something like this
```diff
-looking up slack user by email mail@example.org
-{ "email": "mail@example.org" }
-will notify #mock_id
+will notify #id[mail@example.org]
```